### ISSUE TITLE
Missing dependency and fatal error

### ISSUE THREAD

**Issue by adri326** [bigH/git-fuzzy#13]
I'm running Arch Linux, pretty up-to-date and fairly standard, running zsh and installed fzf through the community packages.

I had to install `bc` as a dependency, which wasn't mentioned anywhere.
Additionally, I cannot run `git-fuzzy status` on any of the repositories I have cloned (including this one).

On the repos I have committed in, the following error shows up, and `git-fuzzy` returns 1:

```
fatal: ref refs/remotes/origin/HEAD is not a symbolic ref
```

On other repos, it just returns 1.

**Comment by leighmcculloch** [comment]
I'm seeing the same error. I'm running Debian Buster. I'm using fzf 0.21.1 though, not the Debian fzf. I'm using the Debian package for bc though.

**Comment by austinpray** [comment]
I'm running git-fuzzy @ de1c13b8a5664541254278b7a6982b558847f20c and I am running into the same thing as the OP.

I am on a brand-new Arch linux install as well. I installed `bc` but `git-fuzzy status` still exits with code 1.

![Screen Capture_select-area_20200613210621](https://user-images.githubusercontent.com/2192970/84583093-43214e80-adba-11ea-8a17-a51d7fdcdb80.png)


<details>

<summary>Dump of my installed packages</summary>

```
antibody 6.0.1-1
autoconf 2.69-7
autojump 22.5.3-5
automake 1.16.2-3
aws-cli 1.18.74-1
base 2-2
bat 0.15.4-1
bc 1.07.1-4
binutils 2.34-5
bison 3.6.3-1
compton-conf 0.15.0-1
deepin-desktop-schemas 5.6.0.2-1
deepin-screen-recorder 5.8.0.11-3
deepin-sound-theme 15.10.6-1
dmenu 4.9-1
docker 1:19.03.11-1
docker-compose 1.26.0-1
efibootmgr 17-1
exa 0.9.0-1
fakeroot 1.24-2
feh 3.4.1-1
flatpak 1.7.2-1
flex 2.6.4-3
fluxctl 1.18.0-1
fzf 0.21.1-2
gcc 10.1.0-2
git 2.27.0-1
git-delta 0.1.1-1
git-lfs 2.11.0-2
google-chrome 83.0.4103.97-1
greetd 0.6.1-1
groff 1.22.4-3
grub 2:2.04-7
gvim 8.2.0814-2
i3-gaps 4.18.1-2
i3blocks 1.5-3
i3lock 2.12-1
i3status 2.13-2
ibus 1.5.22+8+gf591381e-1
ibus-mozc 2.23.2815.102-3
imagemagick 7.0.10.18-1
imagemagick-doc 7.0.10.18-1
intel-ucode 20200609-1
j4-dmenu-desktop 2.17-1
jetbrains-toolbox 1.17.7018-1
kitty 0.17.4-1
kitty-terminfo 0.17.4-1
kubectl 1.18.3-1
kubectx 0.9.0-1
linux 5.7.2.arch1-1
linux-firmware 20200519.8ba6fa6-1
m4 1.4.18-3
make 4.3-3
meld 3.20.2-1
mictray 0.2.3-1
networkmanager 1.24.2-1
noto-fonts 20190926-4
noto-fonts-cjk 20190409-2
noto-fonts-emoji 20191016-6
ntp 4.2.8.p14-1
nvidia 440.82-21
nvidia-settings 440.82-1
openssh 8.3p1-1
os-prober 1.77-1
pasystray 0.7.1-1
patch 2.7.6-8
pavucontrol 1:4.0-2
picom 8-2
pkgconf 1.7.3-1
polybar 3.4.3-1
pulseaudio 13.0-3
pulseaudio-alsa 1:1.2.2-2
python-grip 4.5.2-1
redshift 1.12-3
ripgrep 12.1.1-1
ruby 2.7.1-2
ruby-docs 2.7.1-2
rubygems 3.1.3-1
siji-git r23.c691f20-1
spaceship-prompt-git v3.11.2.r14.g56c7e5d-1
starship 0.42.0-1
sudo 1.9.0-1
texinfo 6.7-3
tldr 1.0.0-1
ttf-dejavu-sans-mono-powerline-git r202.e80e3eb-1
ttf-jetbrains-mono 1.0.6-1
ttf-joypixels 5.5.0-3
ttf-symbola 13.00-7
ttf-unifont 13.0.02-1
unzip 6.0-14
which 2.21-5
xclip 0.13-3
xf86-video-vesa 2.4.0-3
xfce4-clipman-plugin 1.6.1-1
xfce4-notifyd 0.6.1-1
xfce4-statusnotifier-plugin 0.2.2-1
xorg-bdftopcf 1.1-2
xorg-docs 1.7.1-3
xorg-font-util 1.3.2-2
xorg-fonts-100dpi 1.0.3-5
xorg-fonts-75dpi 1.0.3-5
xorg-fonts-encodings 1.0.5-2
xorg-iceauth 1.0.8-2
xorg-luit 1.1.1-4
xorg-mkfontscale 1.2.1-2
xorg-server-devel 1.20.8-2
xorg-server-xephyr 1.20.8-2
xorg-server-xnest 1.20.8-2
xorg-server-xvfb 1.20.8-2
xorg-server-xwayland 1.20.8-2
xorg-sessreg 1.1.2-2
xorg-smproxy 1.0.6-3
xorg-x11perf 1.6.1-2
xorg-xauth 1.1-2
xorg-xbacklight 1.2.3-2
xorg-xcmsdb 1.0.5-3
xorg-xcursorgen 1.0.7-2
xorg-xdpyinfo 1.3.2-4
xorg-xdriinfo 1.0.6-2
xorg-xev 1.2.3-2
xorg-xgamma 1.0.6-3
xorg-xhost 1.0.8-2
xorg-xinit 1.4.1-2
xorg-xinput 1.6.3-2
xorg-xkbevd 1.1.4-3
xorg-xkbutils 1.0.4-4
xorg-xkill 1.0.5-2
xorg-xlsatoms 1.1.3-2
xorg-xlsclients 1.1.4-2
xorg-xmodmap 1.0.10-2
xorg-xpr 1.0.5-2
xorg-xprop 1.2.4-2
xorg-xrandr 1.5.1-2
xorg-xrdb 1.2.0-2
xorg-xrefresh 1.0.6-2
xorg-xset 1.2.4-2
xorg-xsetroot 1.1.2-2
xorg-xvinfo 1.1.4-2
xorg-xwd 1.0.7-2
xorg-xwininfo 1.1.5-2
xorg-xwud 1.0.5-2
yay-bin 9.4.7-1
zsa-wally 1.1.1-1
zsh 5.8-1
```

</details>

**Comment by leighmcculloch** [comment]
The issue I was having (`fatal: ref refs/remotes/origin/HEAD is not a symbolic ref`) was I think because I had fiddled with the remotes configured on my clone and a fresh clone of my repo resulted in it working fine.

**Comment by leighmcculloch** [comment]
I opened #17 which I think is a reasonable action in the case the symbolic-ref is missing, but I look forward to hearing from @bigH if they think it is reasonable.

**Comment by claytonrcarter** [comment]
All of my primary repos were causing issuing this warning as well. Short of #17, I "fixed" it by just running `git remote set-head origin master` in each of those to make sure that `origin/HEAD` was pointing to `origin/master`. (Ref https://gist.github.com/libitte/cbde168d26bc5faf9bf9fef648091b42)

**Comment by bigH** [comment]
merged the change. thank you!
